Playbooks/playbook-Detonate_URL_-_Lastline.yml

```diff
@@ -425,5 +472,25 @@ outputs:
 - contextPath: Lastline.Submission.Process.executable.yara_signature_hits
   description: Yara signature of the executable of the process
   type: string
+- contextPath: URL
+  description: URL object
+  type: unknown
+- contextPath: URL.Malicious
+  description: URL Malicious object
+  type: unknown
+- contextPath: File
```

Comment:
does this supposed to be InfoFile?

Reply:
it's a mistake.
no infofile -you can't get the original sample in lastline.

Reply:
InfoFile is for the reports.
File is for the original sample